package.json

```diff
@@ -31,6 +33,7 @@
     "test-sauce": "wct --skip-plugin local --plugin sauce elements/*/test",
     "system-test": "node_modules/mocha/bin/mocha --timeout 500000 --reporter spec",
     "watch": "gulp watch",
+    "postinstall" : "node postinstall.js",
```

Comment:
In `make init' we do an `npm install`. Is this going to run and mess the rest of our development process?